docs/courses/design/brainstorming-datasets.md

```diff
@@ -71,9 +71,13 @@ Bigger datasets are not always better. In many cases, having a dataset that is s
 
 ## Common problems and their solutions
 
-### Using the iris dataset
+### Forbidden datasets
```

Comment:
Consider adding "Titanic".

Reply:
"Forbidden" sounds a little bit gratuitously harsh. It also ought to be restated as a problem (for consistency with the rest of the docs). Maybe "Using overused datasets".

Reply:
I'd like to revamp all of this actually. There are too many sections at the moment and the formatting doesn't make the division between sections super clear (the font sizes are not _that_ different). I think that "Using overused datasets" doesn't convey the idea of "Don't use these", but rather "use at your own peril".